src/components/Home/Aditya/Week3/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Counter from './components/counter/Counter';
+import Form from './components/form/Form';
+import { Assignment3Div } from './styles';
+import Status from './components/status/Status';
+
+const Week3: React.FC = () => {
+  return (
+    <Assignment3Div>
+      <div id='counter' className='relative'>
+        <Counter></Counter>
```

Comment:
You can use `<Counter />`

Reply:
@aditya-s-19  please check across the project